rasa/core/actions/forms.py

```diff
@@ -202,7 +202,7 @@ def get_entity_value_for_slot(
         if isinstance(tracker.slots.get(slot_to_be_filled), ListSlot):
             return value
 
-        if len(value) == 0:
+        if not value:
```

Comment:
Function `FormAction.get_entity_value_for_slot` refactored with the following changes:

- Simplify sequence comparison ([simplify-len-comparison](https://docs.sourcery.ai/refactorings/simplify-len-comparison/))